syntax = "proto3";

package unary;

service Unary{
  // A simple RPC.
  //
  // Sends fields of new town to server.
 rpc GetServerResponse(Message) returns (MessageResponse) {}

}

message Message{
 string district = 1;
 string town = 2;
 string town_type = 3;
 int32 population = 4;
}

message MessageResponse{
 string message = 1;
 bool received = 2;
}